test/IDE/complete_in_accessors.swift

```diff
@@ -247,18 +251,18 @@ willSet {
 }
 
 var globalAccessorDidSet1: Int {
-  didSet(oldValue) {
```

Comment:
We should have at least one test using the explicit form - are there any left here?

Reply:
Oh I thought we have test cases with explicit oldValue because I just reverted https://github.com/apple/swift/pull/26632/files#diff-e05169281e309d1d8b64b08a525071ed change. But apparently we don't have any. I will add some.

Reply:
Done.